ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -66,13 +66,23 @@
 
   /**
    * The percentage of {@link RouterConfig#routerScalingUnitMaxConnectionsPerPortSsl} or
-   * {@link RouterConfig#routerScalingUnitMaxConnectionsPerPortPlainText} to warm up in the startup.
+   * {@link RouterConfig#routerScalingUnitMaxConnectionsPerPortPlainText} to warm up for data nodes in the local
+   * datacenter during startup.
    * {@link RouterConfig#routerConnectionsWarmUpTimeoutMs} may need to be adjusted.
    */
-  @Config("router.connections.warm.up.percentage.per.port")
+  @Config("router.connections.local.dc.warm.up.percentage")
   @Default("25")
-  public final int routerConnectionsWarmUpPercentagePerPort;
+  public final int routerConnectionsLocalDcWarmUpPercentage;
 
+  /**
+   * The percentage of {@link RouterConfig#routerScalingUnitMaxConnectionsPerPortSsl} or
+   * {@link RouterConfig#routerScalingUnitMaxConnectionsPerPortPlainText} to warm up for data nodes in the local
```

Comment:
The comments for routerConnectionsRemoteDcWarmUpPercentage need to be changed perhaps!

Reply:
fixed!